docker-compose.yml

```diff
@@ -14,6 +14,11 @@ services:
     #      - /dev/gpiomem:/dev/gpiomem
     #        /dev/ttyUSB0:/dev/ttyUSB0
     #        /dev/ttyUSB1:/dev/ttyUSB1
+    #        /dev/ttyACM0:/dev/ttyACM0
+    volumes:
+      - ./docker/microlab-docker.ini:/etc/microlab/microlab.ini
+      - ./backend:/app
+      
```

Comment:
/var/lib/microlab should also be mapped somewhere (./docker/data maybe? and then add that to .gitignore) so any recipes that get uploaded persist between restarts